quill-core/src/main/scala/io/getquill/context/mirror/MirrorContextMacro.scala

```diff
@@ -17,25 +15,30 @@ class MirrorContextMacro(val c: MacroContext) extends ContextMacro {
   override protected def prepare(ast: Ast, params: List[Ident]) =
     IsDynamic(ast) match {
       case false =>
+        val returning = CollectAst(ast) {
+          case Returning(_, property) => property
```

Comment:
yes, it is done in sql's Prepare, but not sure if it's on the right place. Maybe I should move it to the RenameProperties.